migrate/migrations/20190807210857-set-default-system-limits.sql

```diff
@@ -0,0 +1,15 @@
+-- +migrate Up
+INSERT INTO config_limits (id, max)
+    VALUES 
+        ('notification_rules_per_user', 15),
+        ('contact_methods_per_user', 10),
```

Comment:
These are indented with non-breaking spaces (`0xa0c2`) somehow, which is causing a syntax error. 

To be consistent with other migrations, remove the indent from `VALUES` and then indent these with 4 spaces (`0x20`)

Double check with `hexdump -C` to see the hex codes to double check, but a simple `make regendb` should tell you if it's fixed.